src/components/Editor/SessionList.vue

```diff
@@ -21,7 +21,10 @@
   -->
 
 <template>
-	<Popover class="session-list" placement="bottom">
+	<Popover class="session-list"
+		placement="bottom"
+		@after-show="applyFocus"
+		@after-hide="revertFocus">
```

Comment:
Maybe we can additionally catch tab/escape and prevent the default event handling and instead just revert the focus.